### ISSUE TITLE
Live Demo available at s.citvy.com (Nov 2020) - works

### ISSUE THREAD

**Issue by MindfulMe** [yTakkar/React-Instagram-Clone-2.0#83]
https://s.citvy.com/

**Comment by MindfulMe** [comment]
We will make our version open-source pretty soon as we fix most bugs and do the mobile version

**Comment by AlirezaAsadi96** [comment]
Can you tell when your version released?
I really need the mobile version.
Thanks

**Comment by MindfulMe** [comment]
@AlirezaAsadi96 Hi, we use the latest one with our custom updates. We are currently planning to roll out an English version that is why such a big delay.

**Comment by MindfulMe** [comment]
@AlirezaAsadi96 feel free to reach us at lampgram@citvy.com if you want to get the exact code right now.

**Comment by B0bbyB0livia** [comment]
> https://s.citvy.com/

this is very nice. how did you make it responsive?

**Comment by MindfulMe** [comment]
Yes, it is responsive. Releasing today, it has some additional functionality tied to a real estate platform. Feel free to submit a bug report or fix request to our repository at https://github.com/citvy/citvy-network

**Comment by MindfulMe** [comment]
It is open source https://github.com/citvy/citvy-network

**Comment by davra47** [comment]
@MindfulMe do you still have your updated version available? I would be very interested, but the link you posted is no longer working unfortunately.

**Comment by MindfulMe** [comment]
@davra47 I will make in a couple of days an open-source version
